README.md

```diff
@@ -36,3 +36,28 @@ Flags:
 
 Use "vault-unsealer [command] --help" for more information about a command.
 ```
+
+## How to setup vault-unsealer via AWS KMS and SSM
+
+[Instruction on existing and new vaults for unsealing vault using KMS and SSM](docs/aws-kms-ssm.md)
+
+## Build from source
+
+```bash
+git clone https://github.com/jetstack/vault-unsealer.git
+```
+
+```bash
+cd vault-unsealer
+export GOPATH=`pwd`
+go get github.com/jetstack/vault-unsealer/cmd
+export CI_COMMIT_TAG=<version>
+export CI_COMMIT_SHA=$(git rev-parse HEAD)
+make build
+```
```

Comment:
To be honest not too sure why you would need  more than that(in a recent go version 1.8+):

```
go get github.com/jetstack/vault-unsealer
make -C $(go env GOPATH)/src/github.com/jetstack/vault-unsealer build
```

Reply:
You're right it was go 1.6 at the time so I'm gonna update the readme build section

your instructions works

thanks!

Reply:
@simonswine round++